src/worker.js

```diff
@@ -1,9 +1,41 @@
+"use scrict"
+
 // The SW will be shutdown when not in use to save memory,
 // be aware that any global state is likely to disappear
 console.log("SW startup");
 
+let CACHE_VERSION = 1;
```

Comment:
This is my first pass at versioning. This number is used to create new caches for each increment of this value. Other values are deleted to make room for it.